modules/rdma-configuring-the-nmstate-operator.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-configuring-the-nmstate-operator_{context}"]
+
+= Configuring the NMState Operator
```

Comment:
@wabouhamad Is this section `Configuring the NMState Operator` needed or can it be removed?

Reply:
This section has been removed.